lib/action.rb

```diff
@@ -23,7 +23,7 @@ def check_version
       description = 'Version is changed'
     else
       state = 'failure'
-      description = 'Branch version is not changed'
+      description = "Branch version is not changed in #{file_path}"
```

Comment:
Could we say `Gem version`? (As I think this just works for gems currently?). As I think branch version is a bit of a weird term 😅